datadog_checks_base/datadog_checks/base/__about__.py

```diff
@@ -1,4 +1,4 @@
 # (C) Datadog, Inc. 2018-present
 # All rights reserved
-# Licensed under a 3-clause BSD style license (see LICENSE)
+# Licensed under a 3-clause BSD style license (see LICENSE) t
```

Comment:
Is this intended?

Reply:
I added this to trigger the CI for every integration to run in parallel rather than all run within a single job. Once the dependencies are fixed I'll remove this extra comment.